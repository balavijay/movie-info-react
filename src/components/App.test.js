import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe ("App Component " , () => {
   const  app = shallow(<App />);
    test("Component rendered" , () => {
        expect(app).toMatchSnapshot();
    })

    test("Header component" , () => {
        expect(app.find('Header').exists()).toBe(true);
    });
 
});