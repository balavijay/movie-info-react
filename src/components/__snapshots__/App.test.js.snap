// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component  Component rendered 1`] = `ShallowWrapper {}`;
